src/server.ts

```diff
@@ -0,0 +1,21 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import awsServerlessExpress from 'aws-serverless-express';
+import express from 'express';
+import { getNewsletters } from './api/newsletters';
+
+const app = express();
+app.use(express.json({ limit: '50mb' }));
+
```

Comment:
Have we captured tickets or details for the remaining steps for things like logging or error handling? This could also just be a todo list somewhere but just so that we don't forget to come back to it